oracle-price-monitor/src/autotask.js

```diff
@@ -0,0 +1,270 @@
+    symbol = await underlyingTokenContract.symbol();
+  }
+  return { decimals, symbol };
+}
+
+function formatAmountString(amount, decimals) {
+  console.log("amount here", amount)
+  const amountBN = ethers.BigNumber.from(amount);
+  const divisorBN = ethers.BigNumber.from(10).pow(decimals);
+
+  // the ethers BigNumber implementation will discard the decimal
+  // portion of the value when we perform the division
+  let resultString = amountBN.toString();
+  if (resultString.length <= decimals) {
+    resultString = `0.${'0'.repeat(decimals - resultString.length)}${resultString[0]}`;
+  } else {
+    resultString = amountBN.div(divisorBN).toString();
+  }
+
+  // format the number to have comma separators for powers of 1000
+  const internationalNumberFormat = new Intl.NumberFormat('en-US');
+  return internationalNumberFormat.format(resultString);
+}
+
+async function createDiscordMessage(reporterPrice, cTokenAddress, transactionHash, provider) {
+  const { decimals, symbol } = await getDecimalsAndSymbol(cTokenAddress, provider);
+
+  const amountString = formatAmountString(reporterPrice, decimals);
+
+  console.log("amount string here", amountString)
+
+  // // construct the Etherscan transaction link
+  const etherscanLink = `[TX](<https://etherscan.io/tx/${transactionHash}>)`;
+
+  return `${etherscanLink} 🚫 reported price of **${amountString}** for **${symbol}** was rejected`;
+}
+
+// post to discord
+async function postToDiscord(url, message) {
+  let response;
+  try {
+    // perform the POST request
+    response = needle.post(url, {content: message}, {json: true} )
+  } catch (error) {
+    // is this a "too many requests" error (HTTP status 429)
+    if (error.response && error.response.status === 429) {
+      // the request was made and a response was received
+      // try again after waiting 5 seconds
+      // eslint-disable-next-line no-promise-executor-return
+      const promise = new Promise((resolve) => setTimeout(resolve, 5000));
+      await promise;
+      response = needle.post(url, {content: message}, {json: true} )
+    } else {
+      // re-throw the error if it's not from a 429 status
+      throw error;
+    }
+  }
+
+  return response;
+}
+
+async function getFortaAlerts(agentId, transactionHash) {
+  const headers = {
+    'content-type': 'application/json',
+  };
+
+  const graphqlQuery = {
+    operationName: 'recentAlerts',
+    query: `query recentAlerts($input: AlertsInput) {
+      alerts(input: $input) {
+        pageInfo {
+          hasNextPage
+          endCursor {
+            alertId
+            blockNumber
+          }
+        }
+        alerts {
+          createdAt
+          name
+          protocol
+          findingType
+          hash
+          source {
+            transactionHash
+            block {
+              number
+              chainId
+            }
+            agent {
+              id
+            }
+          }
+          severity
+		  metadata
+		  description
```

Comment:
```suggestion
      metadata
      description
```
Don't mix tabs with spaces.